### PR TITLE
[NIFI-5192] allow expression language in Schema File property

### PR DESCRIPTION
**Pull request by bartektartanus**

### CODE REVIEW

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateXml.java [hunk 1]
```diff
@@ -72,6 +72,7 @@
             .name("Schema File")
             .description("The path to the Schema file that is to be used for validation")
             .required(true)
+            .expressionLanguageSupported(true)
```
Comment (MikeThomsen) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateXml.java]: This should use the new `ExpressionLanguageScope` type instead.

**Comment by MikeThomsen** [comment]
@bartektartanus It fails `TestValidateXml`. If you run those tests, you'll see that your update doesn't include the `evaluateExpressionLanguage` calls that are needed to pass validation. So please run those tests and you'll see where you need to add some extra fixes to flesh this out.

**Comment by mattyb149** [comment]
I took the liberty of adding the missing evaluateAttributeExpressions() call, the unit tests pass at that point but if Expression Language is present, then we can't count on the File Exists Validator to catch missing schema files. I added a check for file.exists() and unit tests for valid and invalid (with EL). Feel free to cherry-pick https://github.com/mattyb149/nifi/commit/a452382bed64cd4ddda69711e7382829726503ee into your branch and push back up here for review, or @MikeThomsen if you're cool with it you could just cherry pick his and my commits from https://github.com/mattyb149/nifi/tree/NIFI-5192 before merge.

**Comment by mattyb149** [comment]
I added my commit, rebased, and squashed, unit tests and contrib-check passes. Thanks for the improvement! Merging to master

**Comment by mattyb149** [comment]
@bartektartanus I forgot to add the magic string to the commit that automatically closes this PR, do you mind closing it? It has been merged to master, thanks again!

**Pull request closed without merging**